gulpfile.babel.js

```diff
@@ -267,7 +267,6 @@ gulp.task('metadata', () => {
 gulp.task('default', ['clean'], cb => {
   runSequence(
     ['styles', 'styles-grid'],
-    ['scripts'],
```

Comment:
If we are sure that `gulp all`, `gulp` and `gulp serve` (including watches) still work reliably, I’m fine with merging this in as is.